src/hardware/sblaster.cpp

```diff
@@ -3380,7 +3392,17 @@ void init_sblaster_dosbox_settings(Section_prop& secprop)
 	pint->Set_help("The High DMA channel of the Sound Blaster 16 (5 by default).");
 
 	auto pbool = secprop.Add_bool("sbmixer", when_idle, true);
-	pbool->Set_help("Allow the Sound Blaster mixer to modify the DOSBox mixer (enabled by default).");
+	pbool->Set_help(
+	        "Allow the Sound Blaster mixer to modify volume levels (enabled by default).\n"
+	        "Sound Blaster Pro 1 and later cards allow programs to set the volume of the\n"
+	        "digital audio (DAC), FM synth, and CD Audio output. These correspond to the\n"
+	        "SB, OPL, and CDAUDIO DOSBox mixer channels, respectively.\n"
+	        "  on:   The final level of the above channels is a combination of the volume\n"
+	        "        set by the program, and the volume set in the DOSBox mixer.\n"
+	        "  off:  Only the DOSBox mixer determines the volume of these channels.\n"
+	        "Note: Some games change the volume levels dynamically (e.g., lower the FM music\n"
+	        "      volume when speech is playing); it's best to leave 'sbmixer' enabled for\n"
+	        "      such games.");
```

Comment:
You've been doing a great job at improving these config docs lately :+1:

Reply:
Thanks man, I'm glad you noticed 😄 Well, I'm the doco guy too; I should really try to get the User Manual in a releasable state this year too.